pkg/operation/botanist/component/istio/istiod.go

```diff
@@ -17,22 +17,34 @@ package istio
 import (
 	"context"
 	"path/filepath"
+	"time"
 
-	"github.com/gardener/gardener/pkg/client/kubernetes"
+	"github.com/gardener/gardener/pkg/chartrenderer"
+	"github.com/gardener/gardener/pkg/features"
+	gardenletfeatures "github.com/gardener/gardener/pkg/gardenlet/features"
 	"github.com/gardener/gardener/pkg/operation/botanist/component"
 	kutil "github.com/gardener/gardener/pkg/utils/kubernetes"
+	"github.com/gardener/gardener/pkg/utils/managedresources"
 
+	networkingv1alpha3 "istio.io/client-go/pkg/apis/networking/v1alpha3"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	crclient "sigs.k8s.io/controller-runtime/pkg/client"
 )
 
+const (
+	// ManagedResourceName is the name of the ManagedResource containing the resource specifications.
+	ManagedResourceName = "istio"
```

Comment:
Can we add this to https://github.com/gardener/gardener/blob/f08eff4f443dd466659dbe7a4e2271d9d4980640/pkg/operation/care/seed_health.go#L40-L47 now?

Reply:
Istio is deployed if feature gate is enabled, can we keep it required MangedResource I guess no?

Reply:
Yeah, we should conditionally add it only if the feature gate is enabled.